setup.py

```diff
@@ -36,6 +43,8 @@
 setup(name='freqtrade',
       version=__version__,
       description='Crypto Trading Bot',
+      long_description=readme_long,
+      long_description_content_type="text/markdown",
       url='https://github.com/freqtrade/freqtrade',
       author='gcarq and contributors',
       author_email='michael.egger@tsn.at',
```

Comment:
This should probably be updated.

author='Freqtrade Team', relevant common email...

If @gcarq don't mind...

(May be done in a sep. PR)

Reply:
Sure, go for it.